doc/index.rst

```diff
@@ -19,7 +19,7 @@ In addition to the library itself, this NCS add-on contains a :ref:`sample <sb_f
 
 
 .. important::
-   As part of Nordics certification with SoftBank Mobile, this Softbank FOTA library **must** be integrated with your application, unless otherwise agreed with you local Nordic or Softbank representatives.
+   As part of Nordics certification with SoftBank Mobile, this SoftBank FOTA library **must** be integrated with your application, unless otherwise agreed with your local Nordic or Softbank representatives.
```

Comment:
Maybe something like: "To conform with the SoftBank certification, this library [...]"? We should probably settle on using either `SoftBank` or `SoftBank Mobile`.

Reply:
Agreed on this wording